README.md

```diff
@@ -1 +1,9 @@
 # nc-time-axis
+
+Support for netcdftime axes in matplotlib.
```

Comment:
Agreed, but it is something! Rationale for me doing this: I looked at the repo and had no idea at all what it did, and it took until this line in the `__init__` for me to get an inking of what this does. 

So I figured that as a temporary measure at least we might as well foreshorten that process!